src/widgets.ts

```diff
@@ -53,6 +53,9 @@ export abstract class Widget<
 
     this.messageCallback = (event) => {
       if (event.data.mx) {
+        if (event.data.type === "mx/client/oauthRedirect") {
```

Comment:
We can use `PostMessageTypes.ClientOAuthComplete` instead of the string here.

Reply:
I thought I already responded, but I'll update it when I release the new version of the definitions.

Reply:
This should be updated now.